### DISCUSSION TITLE
Upload tracker

### DISCUSSION THREAD

**Discussion by HaveSpacesuit** [iTwin/iTwinUI#666] [General]
Are there plans to develop a generic upload tracker as documented in the [iTwin UX](https://user-images.githubusercontent.com/9084735/171260704-0c803b64-d025-41b0-8709-a817536cbfc4.png)? I'm getting ready to put one together for an app, but if there is going to be a standardized version it could wait.

**Comment by mayank99** [comment]
Hi @HaveSpacesuit, we haven't planned to work on an upload tracker component because no one has requested it yet. I've created an issue for you to follow (https://github.com/iTwin/iTwinUI/issues/667) but I do not think we are going to prioritize it just yet, so if you come up with your own, you are welcome to share it here.
